### ISSUE TITLE
Alt text missing - Pods

### ISSUE THREAD

**Issue by spencercheng** [BYU-ODH/wp-humanities-byu#461]
- [x] Alt-text for the cards on Projects are missing. (Projects Card Template)
- [x] Alt-text for each individual card is missing, including images in the simple light box. (Single Project Page Template)
